R/run.R

```diff
@@ -20,16 +20,68 @@ run.default <- function(x, ...) {
 }
 
 
+#' @param publish Flag for whether to publish results to a Conbench server. See
+#' "Environment Variables" section for how to specify server details. Requires
+#' the benchconnect CLI is installed; see [install_benchconnect()].
```

Comment:
This should autolink correctly, yeah? But regardless we want code formatting there

```suggestion
#' the benchconnect CLI is installed; see [`install_benchconnect`].
```

Reply:
It does already; this is one of the automagics of the markdown update:
<img width="614" alt="image" src="https://user-images.githubusercontent.com/1693477/213029720-1bfbd33a-98ce-4e5d-80e2-0c342d517e0e.png">